env_manager/backends/conda_like_interface.py

```diff
@@ -214,12 +226,20 @@ def list_packages(self):
 
         for package in result_lines[skip_lines:-1]:
             package_info = package.split()
+            package_name = package_info[0]
+            package_build = None if len(package_info) <= 2 else package_info[2]
+            package_channel = None if len(package_info) <= 3 else package_info[3]
             formatted_package = dict(
-                name=package_info[0],
+                name=package_name,
                 version=package_info[1],
-                build=(None if len(package_info) <= 2 else package_info[2]),
-                channel=(None if len(package_info) <= 3 else package_info[3]),
+                build=package_build,
+                channel=package_channel,
             )
+            if package_channel:
+                package_description = self._get_package_info(
+                    package_name, channel=package_channel
+                )["summary"]
```

Comment:
Let's move this `if` to be before the `formatted_package` declaration because it's used to add another key to the dictionary.

That way you could write

```python
formatted_package = dict(
    ...
    description = package_description
)
```